src/components/sections/Info.section.tsx

```diff
@@ -0,0 +1,38 @@
+const InfoSection = () => {
+  //  00092d
```

Comment:
Is this a colour?

Reply:
Yes, and it shouldn't be there. That was temporary! Removed it.